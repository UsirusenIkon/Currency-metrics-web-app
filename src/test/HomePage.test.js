import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';

import Homepage from '../components/HomePage/HomePage';

describe('Homepage', () => {
  it('renders correctly', () => {
    const tree = Renderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <Homepage />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
