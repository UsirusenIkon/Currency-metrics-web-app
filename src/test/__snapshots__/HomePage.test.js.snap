// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<div
  className="home-page"
>
  <div
    className="header flex"
  >
    <span
      className="img-cover"
    >
      <img
        alt="currency and coin"
        className="header-img"
        src="coinCurrency.jpg"
      />
    </span>
    <h1>
      <span
        className="header-text"
      >
        Coin
      </span>
      <span
        className="header-text"
      >
        And
      </span>
      <span
        className="header-text"
      >
        Currency Rates
      </span>
    </h1>
  </div>
  <div
    className="currency-list flex"
  />
</div>
`;
